OrbitGl/TopDownView.h

```diff
@@ -63,7 +63,7 @@ class [[nodiscard]] TopDownInternalNode : public TopDownNode {
 class [[nodiscard]] TopDownFunction : public TopDownInternalNode {
  public:
   explicit TopDownFunction(uint64_t function_absolute_address, std::string function_name,
-                           TopDownNode* parent)
+                           TopDownNode * parent)
```

Comment:
is this necessary? :)

Reply:
Sorry, that was clang-format. I guess now its even worse, but at least correct.